From f03a30d3a36c52f774e5efe078082de297b61fc3 Mon Sep 17 00:00:00 2001
From: AlanYoush <yoush@centec.com>
Date: Tue, 30 May 2023 08:20:52 +0000
Subject: [PATCH] core file prefixed by kdump

---
 debian/kdump-config.in | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/debian/kdump-config.in b/debian/kdump-config.in
index a1c3b4e..ae40492 100755
--- a/debian/kdump-config.in
+++ b/debian/kdump-config.in
@@ -639,8 +639,8 @@ function kdump_save_core()
 {
 	KDUMP_STAMP=`date +"%Y%m%d%H%M"`
 	KDUMP_STAMPDIR=$(define_stampdir $KDUMP_STAMP)
-	KDUMP_CORETEMP="$KDUMP_STAMPDIR/dump-incomplete"
-	KDUMP_COREFILE="$KDUMP_STAMPDIR/dump.$KDUMP_STAMP"
+	KDUMP_CORETEMP="$KDUMP_STAMPDIR/kdump-incomplete"
+	KDUMP_COREFILE="$KDUMP_STAMPDIR/kdump.$KDUMP_STAMP"
 	KDUMP_DMESGFILE="$KDUMP_STAMPDIR/dmesg.$KDUMP_STAMP"
 
 	# If we use NFS, verify that we can mount the FS
@@ -755,8 +755,8 @@ function kdump_save_core_to_ssh()
 	KDUMP_STAMP=`date +"%Y%m%d%H%M"`
 	KDUMP_STAMPDIR=$(define_stampdir $KDUMP_STAMP)
 
-	KDUMP_CORETEMP="$KDUMP_STAMPDIR/dump-incomplete"
-	KDUMP_COREFILE="$KDUMP_STAMPDIR/dump.$KDUMP_STAMP"
+	KDUMP_CORETEMP="$KDUMP_STAMPDIR/kdump-incomplete"
+	KDUMP_COREFILE="$KDUMP_STAMPDIR/kdump.$KDUMP_STAMP"
 	KDUMP_TMPDMESG="/tmp/dmesg.$KDUMP_STAMP"
 	KDUMP_DMESGFILE="$KDUMP_STAMPDIR/dmesg.$KDUMP_STAMP"
 	ERROR=0
-- 
2.25.1

